config/passport-setup.js

```diff
@@ -0,0 +1,77 @@
+const passport = require('passport');
+const GooglePlusTokenStrategy = require('passport-google-plus-token');
+const FacebookTokenStrategy = require('passport-facebook-token');
+const keys = require('./keys');
+const { Add_New_User } = require('../src/queries/Add_New_User');
+const { Find_User } = require('../src/queries/Find_User');
+
+// Google OAuth Strategy
+passport.use('googleToken', new GooglePlusTokenStrategy({
+    clientID: keys.google.clientID,
+    clientSecret: keys.google.clientSecret
+}, async (accessToken, refreshToken, profile, done) => {
```

Comment:
you have the `async` keyword here without using `await` anywhere, and in the promise at the bottom instead is using `await` you used `.then .catch` lets change it to `async/await`